docs/OncoTree-Mapping-Tool.md

```diff
@@ -30,34 +29,164 @@ The list of OncoTree versions available are viewable [here](http://oncotree.mskc
+> In `oncotree_2018_05_01`, `ALL` had two children: `TALL` and `BALL`. On release `oncotree_2018_06_01`, the ALL node was discontinued and the `TALL` node was renamed `TLL` and the `BALL` node was renamed `BLL`. 
+
+**The `ONCOTREE_SUGGESTIONS` column would be shown as follows:**  
+> _ALL -> {TLL, BLL}_  
+
+Ambiguous direct mappings are also checked for addition of more granular nodes; to see how this may affect the `ONCOTREE_SUGGESTIONS` column formatting, please refer to the section on [More Granular Nodes Introduced](#3-more-granular-nodes-introduced).
+
+### 3. No Direct Mappings   
+ No direct mappings occur when the source oncotree code is unrelated to any oncotree code in  the target version. One such possibility is mapping a newly introduced oncotree code backwards in time. In this case, the tool finds the closest set of **neighbors** (e.g parents and children) which are mappable in the target version. The `ONCOTREE_SUGGESTIONS` column returns the set with the keyword **Neighbors** as follows:  
+ 
+ > 'Source Code' -> **Neighbors** {'Code 1', 'Code 2', 'Code 3', ...} &emsp; e.g. _UPA -> Neighbors {BLADDER}_
+ 
+ **Example: Schema describing a case where new OncoTree node UPA cannot be directly mapped backwards to a node.**
+
+![Example 2](https://raw.githubusercontent.com/averyniceday/oncotree/doc-expansion/docs/images/example_2.png)
+
+> In `oncotree_2019_03_01`, `UPA` was added to the oncotree as a child node of `BLADDER`. Because `UPA` did not exist in previous verison `oncotree_2018_05_01` and did not replace any existing node, the tool uses the surrounding nodes when mapping backwards. In this case, the parent node `BLADDER` is returned as the closest match.
+
+**The `ONCOTREE_SUGGESTIONS` column would be shown as follows:**  
+> _UPA -> Neighbors {BLADDER}_
+
+### 4. More Granular Nodes Introduced
+In certain cases, the target version can also introduce nodes with more specfic descriptions. When this occurs, the tool will add the string `more granular choices introduced` to the existing text in the `ONCOTREE_SUGGESTIONS` column as follows:  
+  
+> _'Source Code' -> {'Code 1', ...}, **more granular choices introduced**_  
+> e.g. _TALL -> {TLL}, more granular choices introduced_ 
+
+ **Example: Schema describing a case where OncoTree node TALL is mapped to a node with more granular children**  
+ 
+![Example 3](https://raw.githubusercontent.com/averyniceday/oncotree/doc-expansion/docs/images/example_3.png)
+
+> In `oncotree_2019_03_01`, `TALL` was a leaf node with no children. In release `oncotree_2019_06_01`, `TLL` was introduced as a replacement for `TALL` with additional children `ETPLL` and `NKCLL`. 
+
+**The `ONCOTREE_SUGGESTIONS` column would be shown as follows:** 
+> _TALL -> {TLL}, more granular choices introduced_  
+
+### 5. Invalid Source OncoTree Code  
```

Comment:
we might reorder these to match the order in the output file .. but this can be done in revision